examples/sinatra/views/index.haml

```diff
@@ -7,8 +7,8 @@
     %h1
       Welcome
 
-    %a{ href: "/auth/fishbrain" }
+    %a{ href: "/client/auth/fishbrain" }
       Log in with Fishbrain
 
-    %a{ href: "/auth/cognito-idp" }
+    %a{ href: "/client/auth/cognito-idp" }
```

Comment:
this link is a leftover from some old test and should actually be removed :)

Reply:
Removed